### PR TITLE
Fix list formatting and update screenshot

### PR DESCRIPTION
**Pull request by Cerdtruir**
Fix list formatting and update screenshot according to recommendations on last pr

site demo: http://htmlpreview.github.io/https://github.com/Cerdtruir/odin-recipes/blob/fix_list_formatting/index.html

### CODE REVIEW

**Review by ikraamg — COMMENTED** [review]
Try this:
```css
li {
    color: gray;
    font-weight: bold;
    text-align: left;
    padding: 3px;
}


ul, ol {
  margin: 0;
}
```

to get something like this:

image

**Comment by Cerdtruir** [comment]
Is this better?

**Comment by Cerdtruir** [comment]
@ikraamg

File: style.css [hunk 1]
```diff
@@ -9,10 +9,12 @@ h2 {
...
+  text-align: left;
+  padding: 3px;
 }
-ul {
-  display: inline-block;
-}
+ul,
 ol {
   display: inline-block;
+  padding: 0;
```
Comment (ikraamg) [inline style.css]: This is causing the bullets and numbers on the two lists to be lost.

**Comment by Cerdtruir** [comment]
ok, its fixed @ikraamg 

and it looks like i can search by name and username for @

**Review by ikraamg — APPROVED** [review]
Looks good 👍

**Pull request merged**